### ISSUE TITLE
Reproduce Confusion

### ISSUE THREAD

**Issue by PaulTHong** [dogyoonlee/RSMix#6]
Hey, sorry to bother you. I rerun the code as your instruction in the README. For DGCNN on ModelNet40, the accuracy of RSMix is rather lower than the base version. 
From
python main.py --exp_name=rsmix_dgcnn_1024 --model=dgcnn --num_points=1024 --k=20 --use_sgd=True --beta 1.0 --epochs 500
to
python main.py --exp_name=rsmix_dgcnn_1024 --model=dgcnn --num_points=1024 --k=20 --use_sgd=True --beta 0. --rsmix_prob 0. --epochs 500
Is there anything wrong? Thank for your reply.

**Comment by dogyoonlee** [comment]
@PaulTHong 
Thank you for your interest.
As I understand, is your question is that the performance of DGCNN with the command following is lower than of original DGCNN?
```
python main.py --exp_name=rsmix_dgcnn_1024 --model=dgcnn --num_points=1024 --k=20 --use_sgd=True --beta 0. --rsmix_prob 0. --epochs 500
```

**Comment by PaulTHong** [comment]
Thank for your reply. Yeah, it is. Have you run it again?

**Comment by dogyoonlee** [comment]
@PaulTHong 
Not yet. I guess it can be because of the augmentation parameter.
As I remember, original DGCNN uses the translation and shuffle as default augmentation strategy.
Above run command does not contain any augmentation parameter.
It might the reason of dropped performance.

**Comment by PaulTHong** [comment]
I'm afraid not. You can refer to my first comment, the command is almost the same. I ran the same script, only with two different parameters: "--beta 0. --rsmix_prob 0.".  And I have added two augmentation papameters, but the results are still not as the reported trend.

**Comment by PaulTHong** [comment]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Paul.HT/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Paul.HT/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



  | acc | avg acc
-- | -- | --
rsmix | 0.9307 | 0.8924
rsmix+rscale+rdrop | 0.9335 | 0.9052
base | 0.9307 | 0.8993
base+rscale+rdrop | 0.9352 | 0.901



</body>

</html>

**Comment by dogyoonlee** [comment]
@PaulTHong 
What is your environment setting?

**Comment by PaulTHong** [comment]
Emmm, I think there is no relation with the environment? Just pytorch1.0+ etc. following the requirements.

**Comment by dogyoonlee** [comment]
@PaulTHong 
I'll check the performance later.
Thank you for your question.

**Comment by PaulTHong** [comment]
Thank you very much. Waiting for your good news!

**Comment by PaulTHong** [comment]
Hey, sorry to bother you again. Have you figured it out? Thanks.

**Comment by dogyoonlee** [comment]
No, I can't figure it now yet.

I just trained again with above command you gave and the results were as follows.

============================

                    Model                   Accuracy

      DGCNN                             |   92.7472
      DGCNN + RSMix                     |   93.2739
      DGCNN + rdscale, rddrop           |   93.0713
      DGCNN + RSMix + rdscale, rddrop   |   93.5575

============================

My environment is

python 3.6.13
CUDA 10.2
torch 1.6.0
torch vision 0.7.0

Please try out code again in same environment.
In addition, the results can depend on the seed because of the inherent characteristic of data augmentation.

Thank you for your interest again!

**Comment by PaulTHong** [comment]
Hey, sorry to bother you again. I read your paper again. You said that for ModelNet40 on DGCNN, you used the un-nomalized data? Will it be the reason for mu failure in reproduce? And about your reported results in this issue, do you run the code which is absolutely the same with this repo? And for the augmentation methods, utilized methods in fact （refer to the script command in README） is not the same as reported in paper(rscale+rdrop+rot+jitter etc.). Looking forward to your reply.

**Comment by PaulTHong** [comment]
And how to reflect single-view or multi-views for test in the code repo? Thanks!

**Comment by dogyoonlee** [comment]
@PaulTHong 
As I remember, I used same data on PointNet experiment.
You can run the code with specific augmentation argument that controls which augmentation method apply.
I rerun the code for above experiment data with augmentations I mentioned in our paper and I didn't modify any code on our repository.
Thank you for your interest again.

**Comment by PaulTHong** [comment]
Hey, thank you for your reply again. But I still can't reproduce your reported results in the paper. It's so confusing. I have implemented a new environment following your configs: 
python 3.6.13
CUDA 10.2
torch 1.6.0
torch vision 0.7.0

Then the reruned results are as following:
|                      | best acc | best avg acc | best epoch | seed |
|----------------------|----------|--------------|------------|------|
| base                 | 93.1524  | 89.7698      | 476        | 1    |
| rdscale+rddrop       | 93.2739  | 90.6733      | 489        | 1    |
| rsmix                | 92.9498  | 89.511       | 388        | 1    |
| rsmix+rdscale+rddrop | 93.3955  | 90.6279      | 493        | 1    |

You can see that rsmix still can't exceed the baseline. I am very confused now.
